content/service/exporters/honeycomb/_index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Honeycomb"
 date: 2019-06-10T18:25:17-08:00
-aliases: [/agent/exporters/honeycomb]
+aliases: [/service/exporters/honeycomb]
```

Comment:
Oh oh, please revert this change as I had explained before.

Reply:
yup yup. On it.